projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-row-dimension-content.component.html

```diff
@@ -21,6 +21,7 @@
 
 <ng-template #headerDefaultTemplate let-column>
     <div class='igx-grid__tr--header igx-grid__row-indentation--level-{{getLevel()}}'>
+        <!-- TODO remove inline style if possible -->
```

Comment:
@MayaKirova I am trying to remove all inline styles from the grid templates and since the `flex-shrink: 0` was added by you in [this commit](https://github.com/IgniteUI/igniteui-angular/commit/a2defd78f8c6cb183a91e7fc1b6c1d62e442179e), I wanted to ask you why we need that style.

If the idea was to shrink the icon when it doesn't have value, `flex-shrink` doesn't work. If this is a bug I can log it as a separate issue, otherwise I can remove the inline style.

<img width="994" alt="Screenshot 2023-07-25 at 14 27 09" src="https://github.com/IgniteUI/igniteui-angular/assets/21260963/c230edc8-7bed-46f9-ab45-848155ff3ff1">

Reply:
@desig9stein It's actually for scenarios where the text in some cells is longer than the container, in which case if there's no flex-shrink:0 it will shrink the icon and the texts will no loner be aligned with the other cells. For example:
![image](https://github.com/IgniteUI/igniteui-angular/assets/10397980/5154ba12-94e4-4d2a-bb7a-4f42a68ad163)

Reply:
Ok, I am gonna leave this as an inline style